cmd/kms-rest/startcmd/start.go

```diff
@@ -704,6 +721,28 @@ func startKmsService(params *kmsRestParameters, srv Server) error {
 		handler)
 }
 
+func startMetrics(srv Server, metricsHost string) {
+	if metricsHost == "" {
+		return
+	}
+
+	metricsRouter := mux.NewRouter()
+
+	h := promhttp.HandlerFor(prometheus.DefaultGatherer,
+		promhttp.HandlerOpts{
+			EnableOpenMetrics: true,
+		},
+	)
+
+	metricsRouter.HandleFunc("/metrics", func(w http.ResponseWriter, r *http.Request) {
+		h.ServeHTTP(w, r)
+	})
+
+	if err := srv.ListenAndServe(metricsHost, "", "", metricsRouter); err != nil {
```

Comment:
Can we put a logger to put the metrics port number ? This will help in debugging.